std_msgs/Header header

uint16 service_id
uint16 method_id
uint32 data_length
uint16 client_id
uint16 session_id
uint8 protocol_version
uint8 interface_version
uint8 message_type
uint8 return_code

uint64 crc
uint32 length
uint32 sqc
uint32 data_id
uint32 timestamp_nanoseconds
uint32 timestamp_seconds
uint8 timestamp_sync_status
uint32 event_data_qualifier
uint8 extended_qualifier
uint16 origin_invalid_flags
float32 origin_pos_x
float32 origin_pos_y
float32 origin_pos_z
float32 origin_x_std
float32 origin_y_std
float32 origin_z_std
float32 origin_roll
float32 origin_pitch
float32 origin_yaw
float32 origin_roll_std
float32 origin_pitch_std
float32 origin_yaw_std
uint8 list_invalid_flags
Detection[] detections
float32 list_rad_vel_domain_min
float32 list_rad_vel_domain_max
uint32 list_num_of_detections
float32 aln_azimuth_correction
float32 aln_elevation_correction
uint8 aln_status